src/components/Widgets/NextcloudSystem.vue

```diff
@@ -0,0 +1,229 @@
+        (((sys.mem_total - sys.mem_free) / sys.mem_total) * 100).toFixed(2),
+      );
+      this.memoryGauge.color = this.getMemoryGaugeColor(this.memoryGauge.value);
+    },
+    updateOpcacheMemory() {
+      this.opcache_stats.opcache_hit_rate = parseFloat(
+        this.opcache_stats.opcache_hit_rate,
+      ).toFixed(3);
+      this.opcache.memory_usage.total_memory = (
+        this.opcache.memory_usage.used_memory + this.opcache.memory_usage.free_memory
+      );
+      this.opcache.memory_usage.used_memory_percentage = parseFloat(
+        (this.opcache.memory_usage.used_memory / this.opcache.memory_usage.total_memory) * 100,
+      ).toFixed(1);
+    },
+    updateOpcacheInterned() {
+      this.opcache.interned_strings_usage.total_memory = (
+        this.opcache.interned_strings_usage.used_memory
+        + this.opcache.interned_strings_usage.free_memory
+      );
+      this.opcache.interned_strings_usage.used_memory_percentage = parseFloat(
+        (this.opcache.interned_strings_usage.used_memory
+        / this.opcache.interned_strings_usage.total_memory) * 100,
+      ).toFixed(5);
+    },
+    getMemoryGaugeColor(memPercent) {
+      if (memPercent < 50) return this.getColorRgba('widget-text-color', 0.6);
+      if (memPercent < 60) return '#f6f000';
+      if (memPercent < 80) return '#fca016';
+      if (memPercent < 100) return '#f80363';
+      return '#272f4d';
+    },
+    getMemoryGaugeLabel() {
+      const sys = this.server.nextcloud.system;
+      return `${this.convertBytes((sys.mem_total - sys.mem_free) * 1024, 2, false)} / `
+           + `${this.convertBytes(sys.mem_total * 1024, 2, false)}`;
+    },
+    updateCpuLoad(load) {
+      const chartData = {
+        labels: ['1m', '5m', '15m'],
+        datasets: [{ values: [load[0], load[1], load[2]] }],
+      };
+      const chartTitle = this.tt('load-averages');
+      this.renderCpuLoadChart(chartData, chartTitle);
+    },
+    renderCpuLoadChart(loadBarChartData, chartTitle) {
+      return new this.Chart(`#${this.cpuLoadChartId}`, {
+        title: chartTitle,
+        data: loadBarChartData,
+        type: 'bar',
+        height: 180,
+        colors: [this.getColorRgba('widget-text-color', 0.6)],
+        barOptions: {
+          spaceRatio: 0.2,
+        },
+        tooltipOptions: {
+          formatTooltipY: d => `${d} ${this.tt('tasks')}`,
+        },
+      });
+    },
+  },
+  created() {
+    this.overrideUpdateInterval = 30;
+  },
+  updated() {
+    const load = this.server?.nextcloud?.system?.cpuload;
+    if (load) this.updateCpuLoad(load);
+    const sys = this.server.nextcloud.system;
+    if (sys) this.updateMemoryGauge(sys);
+  },
+};
+</script>
+
+<style scoped lang="scss">
+@import '@/styles/widgets/nextcloud-shared.scss';
+.nextcloud-system-wrapper {
+  div.charts {
+    > div {
+      float: left;
+    }
+    > div:first-child {
+      max-width: 44%;
+      small {
+        font-size: 12px;
+        color: #999999;
```

Comment:
Here, I was trying to match the colour to the small text above the cpu-load-chart, with limited success. 
![Uploading Screenshot 2022-06-19 at 20.14.09.png…]()

Reply:
Maybe
```css
color: var(--widget-text-color);
opacity: var(--dimming-factor);`
```

Not sure if this will match, but at least it can be modified by the current theme, or on the users side if need be.

On a side note, very minor but if possible `rem` is preferred over `px`. Originally this was to respect the users browser preferences / for accessibility, but now it's mostly just for consistency. So this would be `font-side: 0.75rem`.